build.gradle

```diff
@@ -203,7 +203,12 @@ sonarqube {
                            '**/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/GenericAboutToStartOrSubmitCallbackResponse.java',
                            '**/uk/gov/hmcts/reform/finrem/caseorchestration/utils/csv/CaseReference.java',
                            '**/uk/gov/hmcts/reform/finrem/caseorchestration/utils/csv/CaseReferenceKeyValue.java',
-                            '**/uk/gov/hmcts/reform/finrem/caseorchestration/service/SystemUserService.java'
+                            '**/uk/gov/hmcts/reform/finrem/caseorchestration/service/SystemUserService.java',
```

Comment:
This all need fixing. We shouldn't be adding exceptions to sonar without good reason